src/cmd/clone.rs

```diff
@@ -211,5 +212,24 @@ pub fn run(sess: &Session, path: &Path, matches: &ArgMatches) -> Result<()> {
 
     println!("{} dependency added to Bender.local", dep);
 
+    use std::fs::File;
+    let file = File::open(path.join("Bender.lock"))
+        .map_err(|cause| Error::chain(format!("Cannot open lockfile {:?}.", path), cause))?;
```

Comment:
`File::open` errors if `Bender.lock` does not already exist. Is it guaranteed to always exist at this point? If not, would we not want to create it at this point rather than failing?

Reply:
I see, thanks for the explanation!